wallet/src/account/output_cache/mod.rs

```diff
@@ -539,6 +859,37 @@ impl OutputCache {
     }
 }
 
+fn sum_burned_token_amount(
+    outputs: &[TxOutput],
+    token_id: &TokenId,
+) -> Result<Amount, WalletError> {
+    let amount = outputs
+        .iter()
+        .filter_map(|output| match output {
+            TxOutput::Burn(OutputValue::TokenV1(tid, amount)) if tid == token_id => Some(*amount),
+            _ => None,
```

Comment:
it's better to avoid wildcards in matches to get compile error when that enum changes

Reply:
fixed